spec/services/atmosphere/cloud/satisfy_appliance_spec.rb

```diff
@@ -9,16 +9,17 @@
     allow(Atmosphere::Optimizer.instance).to receive(:run)
   end
 
-  let!(:wf) { create(:workflow_appliance_set) }
-  let!(:wf2) { create(:workflow_appliance_set) }
-  let!(:shareable_appl_type) { create(:shareable_appliance_type) }
   let!(:fund) { create(:fund) }
+  let!(:u) { create(:user, funds: [fund]) }
+  let!(:wf) { create(:workflow_appliance_set, user: u) }
+  let!(:wf2) { create(:workflow_appliance_set, user: u) }
+  let!(:shareable_appl_type) { create(:shareable_appliance_type) }
```

Comment:
4 new objects will be created before each spec. Are you sure these 4 elements are needed in every test in this file?